test/byContract/IexecPocoBoost/IexecPocoBoostDelegate.test.ts

```diff
@@ -2326,47 +2147,36 @@ describe('IexecPocoBoost', function () {
                 // Verifications after claiming "claimedTasks" tasks.
                 // Check poco boost balance
                 await expectBalance(
```

Comment:
Because "checking arithmetic" would differ here between balance & frozen. But it is possible though if you prefer, see [Only check intrinsic token balance change diff for each claim](https://github.com/iExecBlockchainComputing/PoCo/pull/150/commits/a35c485326d39415dd0be8352b1b3b540c31affe)